docs/contributing/guidelines-for-writing-upgrade.md

```diff
@@ -19,6 +19,10 @@ Keep in mind that upgrade instructions are written for users that do not underst
 * Write instructions
     * Good example: *"Do this, then that"*
     * Bad example: *"This was done, this was changed"*
+* If there are any upgrading steps that are not vital for project developers but could be helpful for them, 
+    they should be mentioned in the upgrading notes as well and marked as (optional). 
+    The general UPGRADE.md then mentions that the optional steps are not vital, 
```

Comment:
This is not true yet, you need to add the information there :slightly_smiling_face:

Reply:
Yes, atomic commit again. I didn't commit yet. Fixed.